arches_lingo/src/arches_lingo/components/concept/ConceptHeader.vue

```diff
@@ -0,0 +1,179 @@
+import Divider from "primevue/divider";
+import {
+    DEFAULT_ERROR_TOAST_LIFE,
+    ERROR,
+    systemLanguageKey,
+    VIEW,
+} from "@/arches_lingo/constants.ts";
+
+import { fetchLingoResource } from "@/arches_lingo/api.ts";
+import { extractDescriptors } from "@/arches_lingo/utils.ts";
+
+import ResourceInstanceMultiSelectWidget from "@/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/ResourceInstanceMultiSelectWidget.vue";
+
+import type {
+    ConceptHeader,
+    ConceptClassificationStatusAliases,
+    ResourceInstanceResult,
+    DataComponentMode,
+} from "@/arches_lingo/types";
+import type { Language } from "@/arches_vue_utils/types";
+
+const toast = useToast();
+const { $gettext } = useGettext();
+const systemLanguage = inject(systemLanguageKey) as Language;
+
+const concept = ref<ResourceInstanceResult>();
+
+const props = defineProps<{
+    mode: DataComponentMode;
+    sectionTitle: string;
+    componentName: string;
+    graphSlug: string;
+    resourceInstanceId: string;
+    nodegroupAlias: string;
+}>();
+
+const data = ref<ConceptHeader>();
+
+watch(
+    () => concept.value,
+    (newValue) => {
+        const aliased_data = newValue?.aliased_data;
+
+        const name = newValue?.name;
+        const descriptor = extractDescriptors(newValue, systemLanguage);
+        // TODO: get human-readable user name from resource endpoint
+        const principalUser = "Anonymous"; //newValue?.principalUser; // returns userid int
+        // TODO: get human-readable life cycle state from resource endpoint
+        const lifeCycleState = $gettext("Draft");
+        const uri = aliased_data?.uri?.aliased_data?.uri_content?.url;
+        const partOfScheme =
+            aliased_data?.part_of_scheme?.aliased_data?.part_of_scheme;
+        const parentConcepts = (
+            aliased_data?.classification_status || []
+        ).flatMap(
+            (tile: ConceptClassificationStatusAliases) =>
+                tile?.aliased_data
+                    ?.classification_status_ascribed_classification || [],
+        );
+
+        data.value = {
+            name: name,
+            descriptor: descriptor,
+            uri: uri,
+            principalUser: principalUser,
+            lifeCycleState: lifeCycleState,
+            partOfScheme: partOfScheme,
+            parentConcepts: parentConcepts,
+        };
+    },
+);
+
+onMounted(async () => {
+    try {
+        concept.value = await fetchLingoResource(
+            props.graphSlug,
+            props.resourceInstanceId,
+        );
+    } catch (error) {
+        toast.add({
+            severity: ERROR,
+            life: DEFAULT_ERROR_TOAST_LIFE,
+            summary: $gettext("Unable to fetch concept"),
+            detail: error instanceof Error ? error.message : undefined,
+        });
+    }
+});
+</script>
+
+<template>
+    <div class="header-row">
+        <h2>{{ data?.descriptor?.name }} ({{ data?.descriptor?.language }})</h2>
```

Comment:
I think this should instead use gettext interpolation

Reply:
Unless you're asking to parameterize the order of Name & Language, I don't think we should be translating strings from the user - e.g. trying to translate `FISH Archaeological Objects Thesaurus`. That would be handled by storing multiple prefLabels in the desired languages & (hopefully) having the backend provide the 'best' label, which will be included in those API changes we have planned for these components.

Reply:
Ha! touch of overzealousness on my part 😅